python/sglang/srt/layers/attention/triton_ops/decode_attention.py

```diff
@@ -715,6 +719,19 @@ def decode_attention_fwd(
             sm_scale,
             logit_cap,
         )
+    elif _is_hip and get_bool_env_var("CK_MOE"):
+        # ROCM MLA
+        mla_decode_fwd(
```

Comment:
Hey just a heads up this API doesn't match up with the aiter API looks like we are missing qo_indptr

Reply:
and max_seqlen_q